hrjobcontract/api/v3/HRJobContract.php

```diff
@@ -179,3 +179,22 @@ function _civicrm_api3_h_r_job_contract_getlist_output($result, $request) {
   }
   return $output;
 }
+
+/**
+ * HRJobContract.getcurrentcontract API
+ *
+ * @param array $params The accepted params are: contact_id
+ * @return array API result descriptor
+ */
+function civicrm_api3_h_r_job_contract_getcurrentcontract($params) {
+  $contactID = !empty($params['contact_id']) ? (int) $params['contact_id'] : 0;
```

Comment:
Just a question - do we allow go get current contract of user with ID = 0? Maybe just throw an exception saying that contact_id can't be empty?